test/framework/clusterctl/clusterctl_helpers.go

```diff
@@ -127,8 +196,8 @@ func ApplyClusterTemplateAndWait(ctx context.Context, input ApplyClusterTemplate
 
 	Expect(input.ClusterProxy).ToNot(BeNil(), "Invalid argument. input.ClusterProxy can't be nil when calling ApplyClusterTemplateAndWait")
 
-	log.Logf("Creating the workload cluster with name %q using the %q template (Kubernetes %s, %d control-plane machines, %d worker machines)",
-		input.ConfigCluster.ClusterName, valueOrDefault(input.ConfigCluster.Flavor), input.ConfigCluster.KubernetesVersion, *input.ConfigCluster.ControlPlaneMachineCount, *input.ConfigCluster.WorkerMachineCount)
+	log.Logf("Creating the workload cluster with name %q using the %q template (Kubernetes %s, %s control-plane machines, %s worker machines)",
+		input.ConfigCluster.ClusterName, valueOrDefault(input.ConfigCluster.Flavor), input.ConfigCluster.KubernetesVersion, input.ConfigCluster.ControlPlaneMachineCount, input.ConfigCluster.WorkerMachineCount)
```

Comment:
nit: I think we need to make these `%d` again.
This is what I'm seeing in the output currently.
`INFO: Creating the workload cluster with name "clusterctl-upgrade-givf1o" using the "(default)" template (Kubernetes v1.19.1, %!s(*int64=0xc0019840d0) control-plane machines, %!s(*int64=0xc0019840d8) worker machines)`

Reply:
done!